src/App/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+type Props = {
+    children?: any,
+}
+
+export const ErrorBoundary: React.FC<Props> = (props: Props) => {
+    const { children } = props
+
+    const ErrorMessage = () => (
+        <h2>
+            We are sorry, The Roulette just stops :( 
+        </h2>
+    )
+
+    let isEverythingOK  = true
+    return <> { isEverythingOK ? children : <ErrorMessage />}</>
```

Comment:
what about providing error value to the props?

```suggestion
export const ErrorBoundary: React.FC<Props> = (props: Props) => {
    const { children, error } = props

    const ErrorMessage = () => (
        <h2>
            We are sorry, The Roulette just stops :( 
        </h2>
    )
    return error ? <ErrorMessage /> : children;
```

Reply:
implemented 🚀